### DISCUSSION TITLE
Can't see "dependencies" tab in the UI

### DISCUSSION THREAD

**Discussion by pai911** [apache/camel-karavan#1086] [Q&A] [answered]
I can't see "dependencies" tab on a local running Karavan with Docker.

### Startup Command
```
docker run -it --rm  --name karavan \
    --network karavan -p 8080:8080 \
    -e KARAVAN_GIT_INSTALL_GITEA=true \
    -e KARAVAN_MAVEN_CACHE=~/.m2 \
    -e KARAVAN_IMAGE_REGISTRY_INSTALL=true \
    -v /var/run/docker.sock:/var/run/docker.sock \
    ghcr.io/apache/camel-karavan:4.3.0
```

### Wha I see in the browser
<img width="1090" alt="image" src="https://github.com/apache/camel-karavan/assets/6234559/3f792670-a0e2-49e8-8121-e316ee45d030">

### Wha I see in the VS Code extention
<img width="1055" alt="image" src="https://github.com/apache/camel-karavan/assets/6234559/f43f11a2-b477-4799-afeb-10626698ebdd">

It's the same case on the UI of VS Code extention

### Question
Without "dependencies", how do I create a bean that references a bean from a Maven artifact?

Thank you!

**Comment by mgubaidullin** [comment, marked as answer]
Additional dependencies could be added to `application.properties` file:

```
camel.jbang.dependencies=com.foo:acme:1.0
```
